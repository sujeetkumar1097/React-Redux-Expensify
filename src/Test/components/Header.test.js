import React from 'react'
//import ReactShallowRenderer from 'react-test-renderer/shallow'
import Header from '../../components/Header'
import {shallow} from 'enzyme'

test('Header component check', ()=> {
    const wrapper = shallow(<Header/>)
    expect(wrapper).toMatchSnapshot();
    //expect(wrapper.find('h1').text()).toBe('Expensify')
    // const renderer = new ReactShallowRenderer()
    // renderer.render(<Header/>)
    // expect(renderer.getRenderOutput()).toMatchSnapshot()
})