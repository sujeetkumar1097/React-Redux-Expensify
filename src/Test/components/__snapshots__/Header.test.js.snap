// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component check 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/create"
  >
    Create Expense
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
</div>
`;
